### PR TITLE
[REFACTOR] Moves some Cargo-related stuff to master_files

### PR DESCRIPTION
**Pull request by larentoun**
## About The Pull Request

Moves to master_files:
- Supply Packs
- Company Imports ui_act()

## How This Contributes To The Skyrat Roleplay Experience

It's easier to disable and enable back supply packs

## Proof of Testing

Checked order console - hidden
Emagged it - hidden
Ordered `_skyrat` - arrived correctly
Used Company Imports - arrived correctly

## Changelog

:cl:
refactor: Moves cargo supply edits to master_files, company imports ui_act to master_files
/:cl:

### CODE REVIEW

File: modular_skyrat/modules/cargo/code/packs.dm [hunk 1]
```diff
@@ -159,15 +159,11 @@
 	)
 	crate_name = "armor crate"
 
-/datum/supply_pack/security/helmets
+/datum/supply_pack/security/helmets_skyrat
 	name = "Helmets Crate"
-	desc = "Contains three standard-issue brain buckets. Requires Security access to open."
+	desc = "Contains three standard-issue brain buckets."
```
Comment (GoldenAlpharex) [inline modular_skyrat/modules/cargo/code/packs.dm:164]: Why are we removing the string about requiring Security access to open?

**Review by GoldenAlpharex — CHANGES_REQUESTED** [review]

File: modular_skyrat/modules/cargo/code/packs.dm (hunk 1 above)
Comment (larentoun) [reply modular_skyrat/modules/cargo/code/packs.dm:164]: Somewhere in the code it checks if the access is required and will add it to desc. Current version just doubles that "required" text.

File: modular_skyrat/modules/sec_haul/readme.md [hunk 2]
```diff
@@ -16,7 +16,8 @@ Completely overhauls security to something that will hopefully fit us.
 
 ### Master file additions
 
-- N/A
+- `modular_skyrat/master_files/code/modules/cargo/packs/vending_restock.dm`: `var/hidden`
+- `modular_skyrat/master_files/code/modules/cargo/packs/security.dm`: `var/hidden`, `var/cost`
```
Comment (vinylspiders) [inline modular_skyrat/modules/sec_haul/readme.md]: I don't see these anywhere, are these no longer applicable?

File: modular_skyrat/modules/cargo/readme.md [hunk 3]
```diff
@@ -11,22 +11,18 @@ Adds more stuff to cargo.
...
 ### Defines:
 
 - N/A
-<!-- If you needed to add any defines, mention the files you added those defines in -->
 
 ### Master file additions
 
-- N/A
-<!-- Any master file changes you've made to existing master files or if you've added a new master file. Please mark either as #NEW or #CHANGE -->
+- `modular_skyrat/master_files/code/modules/cargo/packs/security.dm`: `var/hidden`
```
Comment (vinylspiders) [inline modular_skyrat/modules/cargo/readme.md]: Same here--I think you might have forgotten to update these when you changed your method of 'removing' the packs from the `hidden` to making use of `special` instead.

**Review by vinylspiders — CHANGES_REQUESTED** [review]
Other than those two things, looks good to me. Merge conflict has been fixed.

File: modular_skyrat/modules/cargo/code/packs.dm (hunk 1 above)
Comment (GoldenAlpharex) [reply modular_skyrat/modules/cargo/code/packs.dm:164]: Alright, cheers!

**Review by GoldenAlpharex — APPROVED** [review]
Will be fine once Vinyl's questions are addressed.

**Review by vinylspiders — APPROVED** [review]

**Pull request merged**